### ISSUE TITLE
Page templates are ignored

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#112]
```
What steps will reproduce the problem?
1. Create a page and assign a page template
2. Update Custom Content Type Manager to Version 0.9.3.2
3. Page templates will only be used when previewing pages but not on published 
pages.

What is the expected output? What do you see instead?
Page is rendered with default instead of custom template.

What version of the plugin are you using?
0.9.3.2
What version of WordPress?
3.1.3

Fixed it after reading 
http://code.google.com/p/wordpress-custom-content-type-manager/issues/detail?id=
111:
Changed condition at line 2630 in /includes/CCTM.php to:
if ( empty($query) || isset($query['pagename']) || isset($query['preview']) || 
isset($query['feed']) || isset($query['page_id'])|| !empty($query['post_type']) 
) {return $query;}


```

Original issue reported on code.google.com by `steven.m...@gmail.com` on 7 Jul 2011 at 9:06

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the fix!  Wow, that's a vulnerable little query WP made me monkey 
with.  I'll get a fix out ASAP.
```

Original comment by `fireproofsocks` on 7 Jul 2011 at 3:40
- Changed state: **InReview**

**Comment by GoogleCodeExporter** [comment]
```
Ok, the fix was released as version 0.9.3.3.
Committed revision 406302.
```

Original comment by `fireproofsocks` on 7 Jul 2011 at 4:24
- Changed state: **Fixed**
